tests/room_timestamp_to_event_test.go

```diff
@@ -1,9 +1,9 @@
-//go:build msc3030
-// +build msc3030
+//go:build !dendrite_blacklist
+// +build !dendrite_blacklist
 
-// This file contains tests for a jump to date API endpoint,
-// currently experimental feature defined by MSC3030, which you can read here:
-// https://github.com/matrix-org/matrix-doc/pull/3030
+// This file contains tests for the `/timestamp_to_event` client and federation API
+// endpoints (also known as *jump to date*). As defined by MSC3030, which you can read
+// here: https://github.com/matrix-org/matrix-doc/pull/3030
 
```

Comment:
Where can `fetchUntilMessagesResponseHas` live to be shared with MSC2716 and `tests/room_timestamp_to_event_test.go` (MSC3030)?

The `msc2716` build tag isn't built for the Synapse workers environment (see https://github.com/matrix-org/synapse/pull/14074) so `fetchUntilMessagesResponseHas` isn't available for `tests/room_timestamp_to_event_test.go` to use

Reply:
I guess we can move `fetchUntilMessagesResponseHas` to this file instead since it will always be available for `msc2716` to pull from ⏩

Reply:
Conversation continued in https://github.com/matrix-org/complement/pull/559#discussion_r1024716899